setup.py

```diff
@@ -43,7 +43,6 @@
     "time_machine==2.15.0",
     "typeguard==4.3.0",
     "xgboost==2.1.1",
-    "dash_bootstrap_components==1.6.0",
```

Comment:
are we sure we don't need this? Aren't we using it in other dashboards?

Reply:
it's a duplicate, that's why a removed it